File: 344.reverse-string.cs
/*
 * @lc app=leetcode id=344 lang=csharp
 *
 * [344] Reverse String
 */

// @lc code=start
public class Solution
{
    public void ReverseString(char[] s)
    {
        int Left = 0;
        int Right = s.Length - 1;

        while (Left < Right)
        {
            char temp = s[Left];
            s[Left] = s[Right];
            s[Right] = temp;
            Left++;
            Right--;
        }
    }
}
// @lc code=end

File: 344.reverse-string.py
#
# @lc app=leetcode id=344 lang=python3
#
# [344] Reverse String
#

# @lc code=start
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        left = 0
        right = len(s) - 1

        while left < right:
            s[left], s[right] = s[right], s[left]
            left += 1
            right -= 1
        
        
# @lc code=end

